srcpkgs/mindustry/template

```diff
@@ -0,0 +1,42 @@
+# Template file for 'mindustry'
+pkgname=mindustry
+version=97
+revision=1
+archs="i686 x86_64"
```

Comment:
with the ~very possibly possibility~ merge of #16101 , there's at least `x86_64-musl` now too. could you make sure?

Reply:
I think this should be `archs=noarch` I can't see any binaries.

Reply:
Done